### ISSUE TITLE
ArgumentException

### ISSUE THREAD

**Issue by Drak0sha13** [agileobjects/ReadableExpressions#102]
I just opened the *options*. The font is not supported.

Visual Studio 16.9.4
AgileObjects.ReadableExpressions.Visualizers.Vs16 4.2.0.0
Windows 7 x64

```Подробная информация об использовании оперативной 
(JIT) отладки вместо данного диалогового 
окна содержится в конце этого сообщения.

************** Текст исключения **************
System.ArgumentException: Шрифт 'Aharoni' не поддерживает стиль 'Regular'.
   в System.Drawing.Font.CreateNativeFont()
   в System.Drawing.Font.Initialize(FontFamily family, Single emSize, FontStyle style, GraphicsUnit unit, Byte gdiCharSet, Boolean gdiVerticalFont)
   в System.Drawing.Font..ctor(FontFamily family, Single emSize)
   в AgileObjects.ReadableExpressions.Visualizers.Dialog.Controls.FontFamilySelector.<>c.<GetFonts>b__3_0(FontFamily ff)
   в System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   в System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   в System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   в AgileObjects.ReadableExpressions.Visualizers.Dialog.Controls.FontFamilySelector.GetFonts()
   в AgileObjects.ReadableExpressions.Visualizers.Dialog.Controls.FontFamilySelector..ctor(VisualizerDialog dialog)
   в AgileObjects.ReadableExpressions.Visualizers.Dialog.Controls.FontSelector..ctor(VisualizerDialog dialog)
   в AgileObjects.ReadableExpressions.Visualizers.Dialog.Controls.VisualizerOptionsMenuItem.Populate()
   в AgileObjects.ReadableExpressions.Visualizers.Dialog.Controls.VisualizerOptionsMenuItem.<>c.<.ctor>b__2_0(Object sender, EventArgs args)
   в System.Windows.Forms.ToolStripDropDownItem.OnDropDownShow(EventArgs e)
   в System.Windows.Forms.ToolStripMenuItem.OnDropDownShow(EventArgs e)
   в System.Windows.Forms.ToolStripDropDownItem.ShowDropDownInternal()
   в System.Windows.Forms.ToolStripDropDownItem.ShowDropDown(Boolean mousePush)
   в System.Windows.Forms.ToolStripMenuItem.OnMouseButtonStateChange(MouseEventArgs e, Boolean isMouseDown)
   в System.Windows.Forms.ToolStripMenuItem.OnMouseDown(MouseEventArgs e)
   в System.Windows.Forms.ToolStripItem.HandleMouseDown(MouseEventArgs e)
   в System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   в System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   в System.Windows.Forms.ToolStrip.OnMouseDown(MouseEventArgs mea)
   в System.Windows.Forms.Control.WmMouseDown(Message& m, MouseButtons button, Int32 clicks)
   в System.Windows.Forms.Control.WndProc(Message& m)
   в System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   в System.Windows.Forms.ToolStrip.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by SteveWilkes** [comment]
Thanks for letting me know about this! Looks like it's [this issue](https://answers.microsoft.com/ru-ru/windows/forum/all/%D1%88%D1%80%D0%B8%D1%84%D1%82-aharoni/563534ac-964d-4e05-a838-42caaff24498?auth=1) - I guess Windows Forms doesn't support all Fonts being loaded into a Font object. I'll sort it out.

**Comment by SteveWilkes** [comment]
This is (or certainly should be) fixed as of v4.4 of the visualizers, which is now available via [VS Marketplace](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1232914.ReadableExpressionsVisualizers).
